examples/eve/next-env.d.ts

```
@@ -1,6 +1,6 @@
 /// <reference types="next" />
 /// <reference types="next/image-types/global" />
-import "./.next/dev/types/routes.d.ts";
+import "./.next/types/routes.d.ts";
```

ESLint

[error] Extra semicolon.

(semi)